README.md

```diff
@@ -5,31 +5,19 @@
 
 # InputStream Adaptive add-on for Kodi
 
-This is an adaptive file addon for kodi's new InputStream Interface.
-
-- this addon is part of the official kodi repository and part of each kodi installation
-- configure the addon by adding URL prefixes wich are allowed to be played by this addon
-- create a .strm file / or addon with passes an url with sets inputstream.adaptive.manifest_type to either "mpd", "ism" or "hls" and open the strm file in kodi
-- or write an addon wich passes an mpd or ism manifest file to kodi
-
-##### Examples:
-1.) mpd dash example with one video and one audio stream
-- Force inputstream.mpd using a property in strm file: #KODIPROP:inputstream=inputstream.adaptive
-- Select the tye of the manifest using a property in strm file: #KODIPROP:inputstream.adaptive.manifest_type=mpd
-- URL to paste into strm file: http://download.tsi.telecom-paristech.fr/gpac/DASH_CONFORMANCE/TelecomParisTech/mp4-live/mp4-live-mpd-AV-BS.mpd
-
-2.) mpd dash example with one video and multiple audio streams
-- Force inputstream.mpd using a Property in strm file: #KODIPROP:inputstream=inputstream.mpd
-- Select the tye of the manifest using a property in strm file: #KODIPROP:inputstream.adaptive.manifest_type=mpd
-- URL to paste into strm file: http://rdmedia.bbc.co.uk/dash/ondemand/testcard/1/client_manifest-events-multilang.mpd
-
-##### Decrypting:
-Decrypting is not implemented. But it is prepared!
-Decrypting takes place in separate decrypter shared libraries, wich are identified by the inputstream.mpd.licensetype listitem property.
-Only one shared decrypter library can be active during playing decrypted media. Building decrypter libraries do not require kodi sources.
-Simply check out the sources of this addon and you are able to build decrypters including full access to existing decrypters implemented in bento4.
-
-##### Bandwidth and resolution:
+This is a [Kodi](https://kodi.tv) input stream add-on wich acts as a demuxer for segmented, multi-bitrate internet streams. The most common streaming protocols such as MPEG-DASH, HLS and Microsoft Smooth Streaming are supported.
```

Comment:
Sorry last comment then I'm all good :)
`wich` typo has survived, should be `which`

Reply:
done